### DISCUSSION TITLE
Youtube: change language for voice search

### DISCUSSION THREAD

**Discussion by Hirishiolo** [AndreyPavlenko/Fermata#310] [Ideas]
When I try to make a voice search in youtube with the mic button, I have to use the language of the interface, but I want to use another language (like official Youtube app). Is it possible?
thanks

**Comment by AndreyPavlenko** [comment]
In the app settings, enable voice control and select the language.

**Reply by AndreyPavlenko** [reply]
Perhaps, it gets the language from your google profile settings.
To check the browser language, open this url in the app's browser - https://gtranslate.io/detect-browser-language

**Comment by Hirishiolo** [comment]
thank you. Long pressing floating button works very well (for italian), but mic searching is only in english (the interface language).

Edit: I noticed that pressing floating button and voice search works very well on phone, but not on car (and in car I don't have "Voice Control" setting)
